src/SFA.DAS.EmployerRequestApprenticeTraining.Web/Views/EmployerRequest/Dashboard.cshtml

```diff
@@ -120,7 +120,7 @@
                                 <p>
                                     <div class="govuk-body govuk-!-margin-bottom-1">Request made on @request.RequestedAt.ToString("d MMM yyyy")</div>
                                     <div class="govuk-hint">
-                                        Expired @request.ExpiryAt.ToString("d MMM yyyy")
+                                        Expired @request.ExpiredAt.Value.ToString("d MMM yyyy")
```

Comment:
The content review sweep was asking for the full month name on the dates, so I expect they will probably want this one with the full month too MMMM